Makefile.in

```diff
@@ -679,7 +679,9 @@ obj:
 # Build the fish program.
 #
 fish: obj/fish.o $(FISH_OBJS) $(EXTRA_PCRE2)
+	echo
 	echo Compiling fish with CXXFLAGS=$(CXXFLAGS)
+	echo ===
```

Comment:
The "===" is inconsistent with similar echo sequences. Personally I prefer surrounding meta comments with "====" (though I typically use ten or twenty equal-signs) so I have no objection to your including such a change but we should be consistent.